docs/commands/lint.md

```diff
@@ -1,16 +1,12 @@
 # `lint`
 
-## Introduction
+## Why you need it
```

Comment:
issue: Changing this heading will make this page inconsistent with all the other pages for commands. Currently, they all follow the same structure: Introduction - Usage - Options - Examples. I believe we want to keep this "predictability" in the way they're internally organized. Do you plan to make this same change on all other command pages?

Reply:
I'll reinstate the current format. Due to my part-time nature, it might take months to sort all of the command pages out so I agree to keep the status quo.